### PR TITLE
Add contribution documentation for Google Summer of Code 2024

### PR DESCRIPTION
**Pull request by Kizjkre**
Thanks for a great GSoC @hoch and @mjwilson-google!

### CODE REVIEW

File: docs/kizjkre-gsoc24.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+![Google Summer of Code logo](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ay9k3nf86inhwwvt32bv.png)
+
+**Organization:** Chromium
+**Projects Overview:**
+- [Web Audio Test and Live Suite](https://github.com/GoogleChromeLabs/web-audio-samples/blob/main/src/tests/playwright/README.md)
+- Rainfly: Audio Worklets Playground
```
Comment (hoch) [inline docs/kizjkre-gsoc24.md:6]: Worklets -> Worklet

To make it consistent within this document

File: docs/kizjkre-gsoc24.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% github https://github.com/GoogleChromeLabs/web-audio-samples/pull/385 %}
+#### Linking the Live Suite to the index page of `web-audio-samples`
+{% github https://github.com/GoogleChromeLabs/web-audio-samples/pull/386 %}
+
+## Rainfly
+Rainfly is an Audio Worklet playground, useful for inspecting, developing, and rapid prototyping of Audio Worklet code.
+### Contributions
+Along with [@terryzfeng](https://github.com/terryzfeng), we created, designed, and developed an MVP of Rainfly using SvelteKit and TailwindCSS.
+### Documentation
+#### Figma med-fi prototype
```
Comment (hoch) [inline docs/kizjkre-gsoc24.md:32]: What's `med-fi`?

**Review by hoch — APPROVED** [review]
LGTM with nits

File: docs/kizjkre-gsoc24.md (hunk 2 above)
Comment (hoch) [reply docs/kizjkre-gsoc24.md:32]: I recommend spelling it out to make it more inclusive. :)

**Review by mjwilson-google — APPROVED** [review]

**Pull request merged**